labellab-flask/api/middleware/logs_decorator.py

```diff
@@ -160,6 +160,13 @@ def wrap(*args, **kwargs):
         issue = find_issue_by_id(issue_id)
         message = f'Issue {issue["title"]} assigned to {user["name"]}'
         category = 'issues'
+     
+      # Comment controller
+      if url.startswith('/api/v1/comment/create/'):
```

Comment:
This log is not getting recorded as the `project_id` is not being set here.

`project_id` is set automatically if the API route contains it. Our create comment API route doesn't have `project_id` in its URL.

So, we have to get `project_id` manually here inside this if statement.
Something like this -

```
issue_id = kwargs.get('issue_id')
# Fetch issue from id and get project_id from there
issue = find_issue_by_id(issue_idi)
project_id = issue['project_id']
```